components/brave_rewards/browser/database/database_recurring_tip.cc

```diff
@@ -0,0 +1,181 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+#include <utility>
+
+#include "base/bind.h"
+#include "base/strings/stringprintf.h"
+#include "brave/components/brave_rewards/browser/database/database_recurring_tip.h"
+#include "sql/statement.h"
+#include "sql/transaction.h"
+
+namespace brave_rewards {
+
+namespace {
+  // TODO(nejczdovc): rename to recurring_tip
```

Comment:
nit: rather than a name we could link to ticket for the work to rename to recurring tip?

Reply:
done